packages/next/src/build/index.ts

```diff
@@ -2334,14 +2333,15 @@ export default async function build(
                   ]
                 : []),
               BUILD_ID_FILE,
-              turboNextBuild ? IS_TURBOPACK_BUILD_FILE : null,
+              SERVER_FILES_MANIFEST,
```

Comment:
We shouldn't make this circular as this isn't needed for standalone mode as we inline the config to the server file we generate here https://github.com/vercel/next.js/blob/canary/packages/next/src/build/utils.ts#L1708